packages/ragbits-document-search/src/ragbits/document_search/ingestion/providers/unstructured/default.py

```diff
@@ -132,20 +137,26 @@ async def process(self, document_meta: DocumentMeta) -> list[Element]:
                 **self.partition_kwargs,
             )
 
-        elements = chunk_elements(elements, **self.chunking_kwargs)
-        return [_to_text_element(element, document_meta) for element in elements]
-
-
-def _to_text_element(element: UnstructuredElement, document_meta: DocumentMeta) -> TextElement:
-    return TextElement(
-        document_meta=document_meta,
-        content=element.text,
-    )
-
-
-def _set_or_raise(name: str, value: Optional[str], env_var: str) -> str:
-    if value is not None:
-        return value
-    if (env_value := os.getenv(env_var)) is None:
-        raise ValueError(f"Either pass {name} argument or set the {env_var} environment variable")
-    return env_value
+        return await self._chunk_and_convert(elements, document_meta, document.local_path)
+
+    async def _chunk_and_convert(
+        self, elements: list[UnstructuredElement], document_meta: DocumentMeta, document_path: Path
+    ) -> list[Element]:
+        if self.__class__ == UnstructuredDefaultProvider:
+            chunked_elements = chunk_elements(elements, **self.chunking_kwargs)
+            return [to_text_element(element, document_meta) for element in chunked_elements]
+        image_elements = [e for e in elements if e.category == "Image"]
```

Comment:
You should probably use Unstructured's `ElementType.Image`  instead of a raw string